### DISCUSSION TITLE
Information on using the `adapter-test` package + jest preset in a custom adapter

### DISCUSSION THREAD

**Discussion by kroy** [nextauthjs/next-auth#9766] [Ideas]
### What is the improvement or update you wish to see?

The AuthJs docs on writing a custom adapter [mention the `adapter-test` package](https://authjs.dev/reference/core/adapters#testing), which can be used to help you test your custom adapter. However, it looks like the [package is private](https://github.com/nextauthjs/next-auth/blob/main/packages/adapter-test/package.json). I also don't see it included in the `@auth/core` package. It'd be great to either clarify how we can use the `adapter-test` package, remove the reference to it, or better yet make it public!

### Is there any context that might help us understand?

Sorry if I'm missing something obvious, or if this is in the wrong Issue section!

### Does the docs page already exist? Please link to it.

https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/adapters.ts#L88-L91

**Comment by balazsorban44** [comment]
Hi, this packages is currently intended for internal usage. However after some polish, we could probably ship it to the community! I'll move this to Ideas, but feel free to open a PR to properly expose this package!

I would love if it had the same TS configuration/setup as other packages (ie: ship as ESM-only, etc.)
